scripts/chce_jenkins.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Jenkins na mikrusowym porcie 
+# Autor: Maciej Loper
+
+status() {
+    echo "[x] $1"
+}
+
+status "dodawanie repozytorium Jenkinsa"
+wget -q -O - https://pkg.jenkins.io/debian-stable/jenkins.io.key | sudo apt-key add -
+sudo sh -c 'echo deb http://pkg.jenkins.io/debian-stable binary/ > /etc/apt/sources.list.d/jenkins.list'
+
+status "aktualizacja repozytoriow"
+sudo apt update
+echo
+
+status "instalacja Jenkinsa i Javy JRE11"
+sudo apt install -y openjdk-11-jre-headless jenkins
+echo
+
+status "poprawki w konfiguracji"
+sudo systemctl stop jenkins
+sudo sed -i 's|HTTP_PORT=8080|HTTP_PORT=80|' /etc/default/jenkins
```

Comment:
```suggestion
sudo sed -i 's|HTTP_PORT=8080|HTTP_PORT=80|' /etc/default/jenkins
sudo sed -i 's|JENKINS_USER=$NAME|JENKINS_USER=root|' /etc/default/jenkins
```

Bez zmiany użytkownika na root, uruchomienie Jenkinsa na porcie 80 będzie niemożliwe. Drugim rozwiązaniem jest uruchomie jako normalny użytkownik (jenkins) oraz dodanie reguł iptables na przekierowanie ruchu z portu 80 na 8080.